interface-definitions/service_monitoring_telegraf.xml.in

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="service">
+    <children>
+      <node name="monitoring">
+        <properties>
+          <help>Monitoring services</help>
+          <priority>1280</priority>
+        </properties>
+        <children>
+          <node name="telegraf" owner="${vyos_conf_scripts_dir}/service_monitoring_telegraf.py">
+            <properties>
+              <help>Service telegraf</help>
+            </properties>
+            <children>
+              <node name="authentication">
+                <properties>
+                  <help>Authentication parameters</help>
+                </properties>
+                <children>
+                  <leafNode name="login">
+                    <properties>
+                      <help>Login</help>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="token">
+                    <properties>
+                      <help>Token</help>
+                    </properties>
+                  </leafNode>
+                </children>
+              </node>
+              <leafNode name="bucket">
+                <properties>
+                  <help>Remote bucket, by default equal to a hostname</help>
+                </properties>
+              </leafNode>
+              <leafNode name="url">
+                <properties>
+                  <help>Remote url</help>
```

Comment:
Maybe design a URL validator and use it here.

Reply:
An example of URL ```https://foo.local:8086```

Reply:
Added regex for URL